pipenv/utils/requirements.py

```diff
@@ -223,7 +223,7 @@ def requirement_from_lockfile(
         if "extras" in package_info
         else ""
     )
-    pip_line = f"{package_name}{extras}=={version}{os_markers}{markers}{hashes}"
```

Comment:
The cause of our version specifiers being too strict, because this method is actually used in more places than just `requirement_from_lockfile` 👎